2025-03-23 09:21:42,710 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:21:42,797 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:21:42,802 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:21:42,802 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:22:41,723 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:22:41,739 INFO: 172.18.0.1 - - [23/Mar/2025 09:22:41] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:22:42,889 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:22:42,893 INFO: 172.18.0.1 - - [23/Mar/2025 09:22:42] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:22:46,011 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:22:46,014 INFO: 172.18.0.1 - - [23/Mar/2025 09:22:46] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:02,787 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:23:02,792 INFO: 172.18.0.1 - - [23/Mar/2025 09:23:02] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:05,824 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:23:05,829 INFO: 172.18.0.1 - - [23/Mar/2025 09:23:05] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:22,828 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:23:22,835 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://172.18.0.3:8000 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:22,835 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:22,836 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:22,909 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:23:22,915 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:22,915 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:53,251 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:23:53,258 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://172.18.0.3:8000 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:53,258 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:53,259 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:53,347 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:23:53,353 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:23:53,353 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:27:10,384 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:27:10,390 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://172.18.0.3:8000 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:27:10,390 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:27:10,391 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:27:10,465 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:27:10,470 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:27:10,471 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:27:20,303 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:27:20,312 INFO: 172.18.0.1 - - [23/Mar/2025 09:27:20] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:28:18,419 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:28:18,424 INFO: 172.18.0.1 - - [23/Mar/2025 09:28:18] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:30:46,130 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:30:46,559 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:30:46,636 INFO: Starting Flask application [in /app/app.py:191]
2025-03-23 09:30:46,642 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:30:46,642 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:31:32,097 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:31:32,140 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:31:32,229 INFO: Starting Flask application [in /app/app.py:182]
2025-03-23 09:31:32,235 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:31:32,235 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:31:59,653 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:31:59,669 INFO: 172.18.0.1 - - [23/Mar/2025 09:31:59] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:00,900 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:00] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:01,770 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:01] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:02,980 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:02] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:05,106 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:05] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:06,408 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:06] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:07,592 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:07] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:08,601 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:08] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:09,983 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:09] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:11,229 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:11] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:12,543 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:12] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:32:13,530 INFO: 172.18.0.1 - - [23/Mar/2025 09:32:13] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:33:49,965 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:33:49,970 INFO: 172.18.0.1 - - [23/Mar/2025 09:33:49] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:34:09,614 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:34:09,621 INFO: 172.18.0.1 - - [23/Mar/2025 09:34:09] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:34:20,521 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:34:20,529 INFO: 172.18.0.1 - - [23/Mar/2025 09:34:20] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:34:22,300 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:34:22,304 INFO: 172.18.0.1 - - [23/Mar/2025 09:34:22] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:34:24,651 INFO: 172.18.0.1 - - [23/Mar/2025 09:34:24] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:34:31,440 INFO: 172.18.0.1 - - [23/Mar/2025 09:34:31] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:34:32,657 INFO: 172.18.0.1 - - [23/Mar/2025 09:34:32] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:34:34,727 INFO: 172.18.0.1 - - [23/Mar/2025 09:34:34] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:07,801 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:36:07,806 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:07] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:10,256 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:10] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:11,463 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:11] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:12,604 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:12] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:14,175 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:14] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:15,454 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:15] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:16,398 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:16] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:17,567 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:17] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:18,445 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:36:18,449 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:18] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:20,864 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:36:20,867 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:20] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:21,984 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:21] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:23,051 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:23] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:23,884 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:23] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:36:24,814 INFO: in test1 [in /app/app.py:45]
2025-03-23 09:36:24,817 INFO: 172.18.0.1 - - [23/Mar/2025 09:36:24] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:12,349 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:12,798 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:12,884 INFO: Starting Flask application [in /app/app.py:188]
2025-03-23 09:38:12,889 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:12,890 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:14,915 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:14,964 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:15,047 INFO: Starting Flask application [in /app/app.py:188]
2025-03-23 09:38:15,053 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:15,054 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:17,078 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:17,127 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:17,206 INFO: Starting Flask application [in /app/app.py:190]
2025-03-23 09:38:17,212 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:17,212 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:18,227 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:18,269 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:18,350 INFO: Starting Flask application [in /app/app.py:192]
2025-03-23 09:38:18,356 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:18,356 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:38:41,090 INFO: in test2 [in /app/app.py:53]
2025-03-23 09:38:41,106 INFO: 172.18.0.1 - - [23/Mar/2025 09:38:41] "GET /test2 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:39:55,666 INFO: in test2 [in /app/app.py:53]
2025-03-23 09:39:55,672 INFO: 172.18.0.1 - - [23/Mar/2025 09:39:55] "GET /test2 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:39:56,758 INFO: 172.18.0.1 - - [23/Mar/2025 09:39:56] "[33mPOST /items HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:40:00,123 INFO: in test2 [in /app/app.py:53]
2025-03-23 09:40:00,125 INFO: 172.18.0.1 - - [23/Mar/2025 09:40:00] "GET /test2 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:40:02,914 INFO: 172.18.0.1 - - [23/Mar/2025 09:40:02] "[33mPOST /items HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:40:45,748 INFO: in test2 [in /app/app.py:53]
2025-03-23 09:40:45,755 INFO: 172.18.0.1 - - [23/Mar/2025 09:40:45] "GET /test2 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:40:47,023 INFO: 172.18.0.1 - - [23/Mar/2025 09:40:47] "[33mPOST /items HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:40:59,319 INFO: in test1 [in /app/app.py:47]
2025-03-23 09:40:59,323 INFO: 172.18.0.1 - - [23/Mar/2025 09:40:59] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:00,654 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:00] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:01,497 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:01] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:02,564 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:02] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:04,212 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:04] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:05,386 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:05] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:06,451 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:06] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:07,252 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:07] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:41:09,494 INFO: 172.18.0.1 - - [23/Mar/2025 09:41:09] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:45:20,780 INFO: Starting Flask application [in /app/app.py:192]
2025-03-23 09:45:20,787 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://172.18.0.3:8000 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:45:20,787 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:45:20,788 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:45:20,864 INFO: Starting Flask application [in /app/app.py:192]
2025-03-23 09:45:20,870 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:45:20,870 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:46:43,281 INFO: 172.18.0.1 - - [23/Mar/2025 09:46:43] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:46:44,022 INFO: 172.18.0.1 - - [23/Mar/2025 09:46:44] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:46:44,778 INFO: 172.18.0.1 - - [23/Mar/2025 09:46:44] "[33mPOST /api/reorder HTTP/1.1[0m" 404 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:22,165 INFO: Accessing index page [in /app/app.py:38]
2025-03-23 09:47:22,175 DEBUG: Found 0 chapters [in /app/app.py:40]
2025-03-23 09:47:22,181 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:22] "GET / HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:24,029 INFO: Accessing index page [in /app/app.py:38]
2025-03-23 09:47:24,032 DEBUG: Found 0 chapters [in /app/app.py:40]
2025-03-23 09:47:24,033 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:24] "GET / HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:25,792 INFO: Adding new chapter: sdf [in /app/app.py:83]
2025-03-23 09:47:25,805 DEBUG: Successfully added chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:90]
2025-03-23 09:47:25,807 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:25] "POST /api/chapters HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:26,991 INFO: Adding new chapter: sdfd [in /app/app.py:83]
2025-03-23 09:47:26,995 DEBUG: Successfully added chapter with ID: 67dfd8aeb1176108762a016c [in /app/app.py:90]
2025-03-23 09:47:26,995 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:26] "POST /api/chapters HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:28,542 INFO: Adding new chapter: gf43 [in /app/app.py:83]
2025-03-23 09:47:28,545 DEBUG: Successfully added chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:90]
2025-03-23 09:47:28,546 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:28] "POST /api/chapters HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:29,738 INFO: Accessing index page [in /app/app.py:38]
2025-03-23 09:47:29,741 DEBUG: Found 3 chapters [in /app/app.py:40]
2025-03-23 09:47:29,743 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:29] "GET / HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:31,111 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:47:31,112 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:31] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:33,203 INFO: Fetching chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:61]
2025-03-23 09:47:33,206 DEBUG: Found chapter: gf43 [in /app/app.py:64]
2025-03-23 09:47:33,210 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:33] "GET /chapter/67dfd8b0b1176108762a016d HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:33,585 INFO: Fetching chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:61]
2025-03-23 09:47:33,588 DEBUG: Found chapter: sdf [in /app/app.py:64]
2025-03-23 09:47:33,590 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:33] "GET /chapter/67dfd8adb1176108762a016b HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:33,970 INFO: Fetching chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:61]
2025-03-23 09:47:33,973 DEBUG: Found chapter: gf43 [in /app/app.py:64]
2025-03-23 09:47:33,975 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:33] "GET /chapter/67dfd8b0b1176108762a016d HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:34,586 INFO: Fetching chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:61]
2025-03-23 09:47:34,588 DEBUG: Found chapter: sdf [in /app/app.py:64]
2025-03-23 09:47:34,589 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:34] "GET /chapter/67dfd8adb1176108762a016b HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:35,038 INFO: Fetching chapter with ID: 67dfd8aeb1176108762a016c [in /app/app.py:61]
2025-03-23 09:47:35,042 DEBUG: Found chapter: sdfd [in /app/app.py:64]
2025-03-23 09:47:35,043 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:35] "GET /chapter/67dfd8aeb1176108762a016c HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:47:35,520 INFO: Fetching chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:61]
2025-03-23 09:47:35,525 DEBUG: Found chapter: sdf [in /app/app.py:64]
2025-03-23 09:47:35,526 INFO: 172.18.0.1 - - [23/Mar/2025 09:47:35] "GET /chapter/67dfd8adb1176108762a016b HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:48:47,474 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:48:47,912 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:48:47,989 INFO: Starting Flask application [in /app/app.py:192]
2025-03-23 09:48:47,995 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:48:47,995 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:49:53,344 INFO: Adding new chapter: xvcx [in /app/app.py:83]
2025-03-23 09:49:53,353 DEBUG: Successfully added chapter with ID: 67dfd94189446266bd31f5c3 [in /app/app.py:90]
2025-03-23 09:49:53,357 INFO: 172.18.0.1 - - [23/Mar/2025 09:49:53] "POST /api/chapters HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:49:54,392 INFO: Accessing index page [in /app/app.py:38]
2025-03-23 09:49:54,395 DEBUG: Found 4 chapters [in /app/app.py:40]
2025-03-23 09:49:54,402 INFO: 172.18.0.1 - - [23/Mar/2025 09:49:54] "GET / HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:49:54,416 INFO: 172.18.0.1 - - [23/Mar/2025 09:49:54] "GET /static/js/sortable.js HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:05,649 INFO: Starting Flask application [in /app/app.py:192]
2025-03-23 09:50:05,656 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://172.18.0.3:8000 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:05,656 INFO: [33mPress CTRL+C to quit[0m [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:05,657 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:05,731 INFO: Starting Flask application [in /app/app.py:192]
2025-03-23 09:50:05,736 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:05,737 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:06,397 INFO: Accessing index page [in /app/app.py:38]
2025-03-23 09:50:06,410 DEBUG: Found 4 chapters [in /app/app.py:40]
2025-03-23 09:50:06,417 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:06] "GET / HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:06,428 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:06] "[36mGET /static/js/sortable.js HTTP/1.1[0m" 304 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:13,666 INFO: Accessing index page [in /app/app.py:38]
2025-03-23 09:50:13,667 DEBUG: Found 4 chapters [in /app/app.py:40]
2025-03-23 09:50:13,668 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:13] "GET / HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:13,676 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:13] "[36mGET /static/js/sortable.js HTTP/1.1[0m" 304 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:14,291 INFO: Fetching chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:61]
2025-03-23 09:50:14,297 DEBUG: Found chapter: gf43 [in /app/app.py:64]
2025-03-23 09:50:14,304 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:14] "GET /chapter/67dfd8b0b1176108762a016d HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:14,754 INFO: Fetching chapter with ID: 67dfd8aeb1176108762a016c [in /app/app.py:61]
2025-03-23 09:50:14,758 DEBUG: Found chapter: sdfd [in /app/app.py:64]
2025-03-23 09:50:14,760 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:14] "GET /chapter/67dfd8aeb1176108762a016c HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:15,174 INFO: Fetching chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:61]
2025-03-23 09:50:15,177 DEBUG: Found chapter: sdf [in /app/app.py:64]
2025-03-23 09:50:15,179 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:15] "GET /chapter/67dfd8adb1176108762a016b HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:15,519 INFO: Fetching chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:61]
2025-03-23 09:50:15,521 DEBUG: Found chapter: gf43 [in /app/app.py:64]
2025-03-23 09:50:15,522 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:15] "GET /chapter/67dfd8b0b1176108762a016d HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:15,886 INFO: Fetching chapter with ID: 67dfd94189446266bd31f5c3 [in /app/app.py:61]
2025-03-23 09:50:15,888 DEBUG: Found chapter: xvcx [in /app/app.py:64]
2025-03-23 09:50:15,889 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:15] "GET /chapter/67dfd94189446266bd31f5c3 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:16,339 INFO: Fetching chapter with ID: 67dfd8aeb1176108762a016c [in /app/app.py:61]
2025-03-23 09:50:16,342 DEBUG: Found chapter: sdfd [in /app/app.py:64]
2025-03-23 09:50:16,343 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:16] "GET /chapter/67dfd8aeb1176108762a016c HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:20,805 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:50:20,807 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:20] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:22,479 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:50:22,480 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:22] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:24,153 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:50:24,153 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:24] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:25,363 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:50:25,364 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:25] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:28,035 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:50:28,036 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:28] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:29,517 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:50:29,518 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:29] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:50:30,504 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:50:30,505 INFO: 172.18.0.1 - - [23/Mar/2025 09:50:30] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:01,121 INFO: in test1 [in /app/app.py:47]
2025-03-23 09:59:01,128 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:01] "GET /test1 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:03,706 INFO: Accessing index page [in /app/app.py:38]
2025-03-23 09:59:03,709 DEBUG: Found 4 chapters [in /app/app.py:40]
2025-03-23 09:59:03,711 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:03] "GET / HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:03,725 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:03] "[36mGET /static/js/sortable.js HTTP/1.1[0m" 304 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:05,852 INFO: Adding new chapter: dfg [in /app/app.py:83]
2025-03-23 09:59:05,855 DEBUG: Successfully added chapter with ID: 67dfdb69917ccfadf2b861ba [in /app/app.py:90]
2025-03-23 09:59:05,859 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:05] "POST /api/chapters HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:39,600 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:59:39,601 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:39] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:40,500 WARNING: Attempted to reorder chapters with no IDs provided [in /app/app.py:101]
2025-03-23 09:59:40,501 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:40] "[31m[1mPOST /api/chapters/reorder HTTP/1.1[0m" 400 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:42,465 INFO: Fetching chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:61]
2025-03-23 09:59:42,468 DEBUG: Found chapter: sdf [in /app/app.py:64]
2025-03-23 09:59:42,470 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:42] "GET /chapter/67dfd8adb1176108762a016b HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:42,865 INFO: Fetching chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:61]
2025-03-23 09:59:42,868 DEBUG: Found chapter: gf43 [in /app/app.py:64]
2025-03-23 09:59:42,870 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:42] "GET /chapter/67dfd8b0b1176108762a016d HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:43,269 INFO: Fetching chapter with ID: 67dfd94189446266bd31f5c3 [in /app/app.py:61]
2025-03-23 09:59:43,272 DEBUG: Found chapter: xvcx [in /app/app.py:64]
2025-03-23 09:59:43,273 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:43] "GET /chapter/67dfd94189446266bd31f5c3 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:43,954 INFO: Fetching chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:61]
2025-03-23 09:59:43,957 DEBUG: Found chapter: sdf [in /app/app.py:64]
2025-03-23 09:59:43,958 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:43] "GET /chapter/67dfd8adb1176108762a016b HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:44,380 INFO: Fetching chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:61]
2025-03-23 09:59:44,383 DEBUG: Found chapter: gf43 [in /app/app.py:64]
2025-03-23 09:59:44,384 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:44] "GET /chapter/67dfd8b0b1176108762a016d HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:44,843 INFO: Fetching chapter with ID: 67dfd94189446266bd31f5c3 [in /app/app.py:61]
2025-03-23 09:59:44,847 DEBUG: Found chapter: xvcx [in /app/app.py:64]
2025-03-23 09:59:44,848 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:44] "GET /chapter/67dfd94189446266bd31f5c3 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:45,311 INFO: Fetching chapter with ID: 67dfd8aeb1176108762a016c [in /app/app.py:61]
2025-03-23 09:59:45,314 DEBUG: Found chapter: sdfd [in /app/app.py:64]
2025-03-23 09:59:45,316 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:45] "GET /chapter/67dfd8aeb1176108762a016c HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:45,664 INFO: Fetching chapter with ID: 67dfdb69917ccfadf2b861ba [in /app/app.py:61]
2025-03-23 09:59:45,667 DEBUG: Found chapter: dfg [in /app/app.py:64]
2025-03-23 09:59:45,668 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:45] "GET /chapter/67dfdb69917ccfadf2b861ba HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:46,060 INFO: Fetching chapter with ID: 67dfd94189446266bd31f5c3 [in /app/app.py:61]
2025-03-23 09:59:46,064 DEBUG: Found chapter: xvcx [in /app/app.py:64]
2025-03-23 09:59:46,066 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:46] "GET /chapter/67dfd94189446266bd31f5c3 HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:46,381 INFO: Fetching chapter with ID: 67dfd8b0b1176108762a016d [in /app/app.py:61]
2025-03-23 09:59:46,384 DEBUG: Found chapter: gf43 [in /app/app.py:64]
2025-03-23 09:59:46,385 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:46] "GET /chapter/67dfd8b0b1176108762a016d HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 09:59:46,714 INFO: Fetching chapter with ID: 67dfd8adb1176108762a016b [in /app/app.py:61]
2025-03-23 09:59:46,718 DEBUG: Found chapter: sdf [in /app/app.py:64]
2025-03-23 09:59:46,720 INFO: 172.18.0.1 - - [23/Mar/2025 09:59:46] "GET /chapter/67dfd8adb1176108762a016b HTTP/1.1" 200 - [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 10:01:20,556 INFO:  * Detected change in '/app/app.py', reloading [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 10:01:20,930 INFO:  * Restarting with stat [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 10:01:21,019 INFO: Starting Flask application [in /app/app.py:243]
2025-03-23 10:01:21,025 WARNING:  * Debugger is active! [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
2025-03-23 10:01:21,025 INFO:  * Debugger PIN: 524-720-992 [in /usr/local/lib/python3.11/site-packages/werkzeug/_internal.py:97]
